apps/web-nomic/src/screens/validator_details/components/blocks/index.tsx

```diff
@@ -0,0 +1,89 @@
+import Tooltip from '@mui/material/Tooltip';
+import Typography from '@mui/material/Typography';
+import useAppTranslation from '@/hooks/useAppTranslation';
+import numeral from 'numeral';
+import { FC } from 'react';
+import Loading from '@/components/loading';
+import AvatarName from '@/components/avatar_name';
+import Box from '@/components/box';
+import Result from '@/components/result';
+import { useProfileRecoil } from '@/recoil/profiles/hooks';
+import { useBlocks } from '@/screens/validator_details/components/blocks/hooks';
+import useStyles from '@/screens/validator_details/components/blocks/styles';
+
+type BlockBoxProps = {
+  i: number;
+  item: ReturnType<typeof useBlocks>['state'][number];
+  state: ReturnType<typeof useBlocks>['state'];
+};
+
+const BlockBox: FC<BlockBoxProps> = ({ i, item, state }) => {
+  const { address, imageUrl, name } = useProfileRecoil(item.proposer);
+  const { t } = useAppTranslation('validators');
+  const { classes, cx } = useStyles();
+  return (
+    <Tooltip
+      key={item.height}
```

Comment:
We don't need to add key in this component (just in the ones inside of the loop below)